ScreenRecorderLib/WWMFResampler.cpp

```diff
@@ -84,6 +84,19 @@ CreateResamplerMFT(
 	RETURN_ON_BAD_HR(spTransformUnk->QueryInterface(IID_PPV_ARGS(&spResamplerProps)));
 	RETURN_ON_BAD_HR(spResamplerProps->SetHalfFilterLength(halfFilterLength));
 
+	/* Not sure we _need_ this (as in, it may be done automatically),
+	 * but I'll keep it around until demonstrated otherwise */
+#if RESAMPLER_NEEDS_LOWPASS_FILTER
+	/* Set up a low-pass filter at half of the 
+	 * output sample rate */
+	CComPtr<IPropertyStore> propStore;
+	RETURN_ON_BAD_HR(spTransformUnk->QueryInterface(IID_PPV_ARGS(&propStore)));
+	PROPVARIANT value;
+	value.vt = VT_R4;
+	value.fltVal = 0.5;
+	RETURN_ON_BAD_HR(propStore->SetValue(MFPKEY_WMRESAMP_LOWPASS_BANDWIDTH, value));
+#endif
+
```

Comment:
As the comment says, I don't think this is needed, but to be honest, I don't think I could write this code again if we did, so I'd rather keep it around.